src/css/blocks/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 20px;
+  height: 16px;
+  background-image: url("../../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-origin: border-box;
+  background-position: 2px center;
+}
+
+.stars--4 > .stars__star:nth-child(n + 5) {
```

Comment:
Better not to increase the specificity, it is enough to write `.stars--4 :nth-child(-n+4)`